### PR TITLE
Update docker container

### PR DESCRIPTION
**Pull request by bkutil**
Update docker container to latest version. Make travis use both influx 0.9 and 0.10 for testing.

### CODE REVIEW

**Comment by bkutil** [comment]
cc: @dmehra 

fixes: #72

File: scripts/docker.sh [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
 NAME="influxdb"
-VER="1"
+VER=$(printenv INFLUX_VERSION)
+
+[ -z "$VER" ] && VER="0.9"
```
Comment (dmehra) [inline scripts/docker.sh]: should that not default to 0.10?

**Comment by dmehra** [comment]
+1 great to see that 0.10 support "just worked"

**Comment by bkutil** [comment]
Will fix the default and merge on green.

**Pull request merged**